lib/Controller/LoginController.php

```diff
@@ -489,23 +489,31 @@ public function code(string $state = '', string $code = '', string $scope = '',
 		}
 
 		// Verify audience
-		$tokenAudience = $idTokenPayload->aud;
-		$providerClientId = $provider->getClientId();
-		if (
-			(is_string($tokenAudience) && $tokenAudience !== $providerClientId)
+		$checkAudience = !isset($oidcSystemConfig['login_validation_audience_check'])
+			|| !in_array($oidcSystemConfig['login_validation_audience_check'], [false, 'false', 0, '0'], true);
```

Comment:
I'd say it is also fine to just support actual boolean values, but fine with me 😁